vllm/attention/backends/habana_attn.py

```diff
@@ -137,10 +139,16 @@ def __init__(
         blocksparse_params: Optional[Dict[str, Any]] = None,
         max_seq_len: int = 4096,
     ) -> None:
+        super(AttentionImpl, self).__init__()
         self.kv_cache_dtype = kv_cache_dtype
         self.num_heads = num_heads
         self.head_size = head_size
         self.scale = float(scale)
+        self.qk_matmul = Matmul()
+        self.softmax = Softmax()
+        self.av_matmul = Matmul()
```

Comment:
is the name av_matmul intended? I know we used kv_matmul previously. If we are using av_matmul we should also make sure the mlperf_inference/code/vllm-static/hqt/step-2-postprocess_measure.py util is using the same layer names.

Reply:
also we should align names with optimum-habana-fork
https://github.com/HabanaAI/optimum-habana-fork/blob/74d5dc8a958666a04cc73a8ba25db0cd65f1a7f4/optimum/habana/transformers/models/llama/modeling_llama.py#L339

Reply:
yes, the name av_matmul intended and we will change accordingly the names in mlperf repo.

Reply:
Aligning names with optimum-habana-fork won't be done now as there is still work to do on how to sync these models or even allow the INC measurement within vLLM.

Reply:
@nirda7 I think there is logic in keeping the names similar in habana attn models, we can consider changing the member names to v_cache and k_cache as they are in OHF.

Reply:
ok - names changed